general/server/media/books.md

```diff
@@ -0,0 +1,40 @@
+---
+uid: server-media-books
+title: Books
+---
+
+# Books
+
+The most common organization scheme for books is separation by Audiobook then by Author.
+
+```
+/Books
+    /Audiobooks
+        /Author
+            Book1.flac
+            Book2.flac
```

Comment:
We also support multiple audio files per book but I'm not sure what the best naming scheme is in that situation.

Reply:
I'm not sure what the best scheme would be. I haven't used audio books too much.
```
/Author
    /Book
         Chapter1.flac
         Chapter2.flac
```

This might work.

Reply:
Were you planning on adding this? I think it looks fine.

Reply:
Added